OSDOCS-10010: Agent basic install procedure (#73553)
openshift/openshift-docs: installing/installing_with_agent_based_installer/installing-with-agent-basic.adoc

:_mod-docs-content-type: ASSEMBLY
[id="installing-with-agent-basic"]
= Installing a cluster
include::_attributes/common-attributes.adoc[]
:context: installing-with-agent-basic

toc::[]

> Use the following procedures to install a basic {product-title} cluster using the Agent-based Installer.

Reviewer: ```suggestion
You can install a basic {product-title} cluster using the Agent-based Installer.
```

> Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​".
- https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

For procedures that include optional customizations you can make while using the Agent-based Installer, see xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-with-agent-based-installer[Installing a cluster with customizations].

[id="prerequisites_{context}"]
== Prerequisites

* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
* You read the documentation on xref:../../installing/overview/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
* If you use a firewall or proxy, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.

[id="installing-ocp-agent_{context}"]
== Installing {product-title} with the Agent-based Installer

> The following procedures deploy a single-node {product-title} in a disconnected environment. You can use these procedures as a basis and modify according to your requirements.

Reviewer: ```suggestion
You can deploy a single-node {product-title} in a disconnected environment and modify configurations according to your requirements.
```

> Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​".
- https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

Author: In this case I think it may be important to keep this phrasing as is, because it's basically a disclaimer that the example configurations used are for SNO in a disconnected environment, and that users need to adjust configurations on their own if they want to install a different cluster topology/environment.
